http.go

```diff
@@ -17,6 +17,18 @@ import (
 	"github.com/valyala/bytebufferpool"
 )
 
+var (
+	requestBodyMaxLimit  = 0
+	responseBodyMaxLimit = 0
+)
+
+// SetBodyLimit set the max body limit in request and response
+// if the body size larger than the limit,it will be released
+func SetBodyLimit(reqBodyMaxLimit, respBodyMaxLimit int) {
```

Comment:
This is very confusing. The name of the function and the documentation makes me think this limits the body sizes I can send or receive.

I would call the method something like `SetBodySizePoolLimit` or `SetMaxBodySizeForPool`.

Reply:
Yeah, the naming here is misleading, I've modified it